app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -266,7 +275,12 @@ class ProfileEditActivityTest {
       verifyTextInDialog(textInDialogId = R.string.profile_edit_delete_dialog_title)
       verifyTextInDialog(textInDialogId = R.string.profile_edit_delete_dialog_message)
       onView(withText(R.string.profile_edit_delete_dialog_negative)).perform(click())
-      onView(withId(R.id.profile_delete_button)).check(matches(isDisplayed()))
+
+      activityTestRule.launchActivity(null)
+      testCoroutineDispatchers.advanceUntilIdle()
```

Comment:
For here & below: do we need advanceUntilIdle() or can we just use runCurrent()?

Reply:
For some reasons `runCurrent()` doesn't work so I've added `times()` just so the test passes.